src/schema/schema.graphql

```diff
@@ -2527,6 +2565,14 @@ input Owner {
 	_content: String
 }
 
+input uploadDocumentId {
```

Comment:
maybe you can rename this to input to `uploadDocument`

Reply:
I have  kept it to uploadDoocumentId because , it has only id parameters in it so have kept that

Reply:
for now it's id but in future we can have more data so at that time we shouldn't be changing type name, better to call uploadDocument